infra/main.tf

```
@@ -128,6 +128,13 @@ resource "google_secret_manager_secret_iam_member" "cloud_func_member" {
   member    = "serviceAccount:${google_service_account.cloud_func_service_account.email}"
 }
 
+resource "google_secret_manager_secret_iam_member" "cloud_func_member" {
```

Can't have 2 resources with the same name
```suggestion
resource "google_secret_manager_secret_iam_member" "cloud_func_rawg_member" {
```